packages/ui/src/ui/components/Dialog/df-dialog-utils.ts

```diff
@@ -116,6 +117,8 @@ export function descriptionToDialogField<T = unknown>(
             return {...common, type: 'path', extras};
         case 'pool':
             return {...common, type: 'pool', extras: {...extras, allowEmpty: true}};
```

Comment:
I'm also want to pass here poolTrees value somehow, but I'm not sure that it is right place. Without poolTrees in extra the suggest will not work properly (will work only with default pool tree).

```
(form) => {
        return {
            poolTrees: form.group_0.pool_trees,
        };
    }
```